### PR TITLE
engine: common: imagelib: fixed loading BMP files with v4/v5 header formats

### PR DESCRIPTION
**Pull request by SNMetamorph**

### CODE REVIEW

File: engine/common/imagelib/img_bmp.c [hunk 1]
```diff
@@ -53,9 +53,9 @@ qboolean Image_LoadBMP( const char *name, const byte *buffer, fs_offset_t filesi
 		return false;
 	}
 
-	if( bhdr.bitmapHeaderSize != 0x28 )
+	if(!( bhdr.bitmapHeaderSize == 40 || bhdr.bitmapHeaderSize == 108 || bhdr.bitmapHeaderSize == 124 ))
 	{
-		Con_DPrintf( S_ERROR "Image_LoadBMP: invalid header size %i\n", bhdr.bitmapHeaderSize );
+		Con_DPrintf( S_WARN "Image_LoadBMP: %s have non-standard header size %i\n", name, bhdr.bitmapHeaderSize );
```
Comment (a1batross) [inline engine/common/imagelib/img_bmp.c:58]: Let's keep it as error. Imagelib refuses to load such files anyway

**Review by a1batross — CHANGES_REQUESTED** [review]

File: engine/common/imagelib/img_bmp.c (hunk 1 above)
Comment (SNMetamorph) [reply engine/common/imagelib/img_bmp.c:58]: Hm, maybe instead just try to load file? It looks like a more versatile solution.

Comment (a1batross) [reply engine/common/imagelib/img_bmp.c:58]: No, it's not a good idea to pass potentially broken images and will lead to random bugs.

**Review by a1batross — CHANGES_REQUESTED** [review]

File: engine/common/imagelib/img_bmp.c (hunk 1 above)
Comment (SNMetamorph) [reply engine/common/imagelib/img_bmp.c]: Ok

**Pull request merged**